src/pre-commit/php_cs_fixer.sh

```diff
@@ -8,3 +8,5 @@ then
 fi
 
 vendor/bin/php-cs-fixer fix --diff
+
+git add --all
```

Comment:
It seems this changes comes from previous PR :) Please, rebase with master branch.